api/search/views.py

```diff
@@ -187,67 +69,84 @@ def search_annotated_articles(main_query, dimensions_json):
 class SearchHelper(object):
     mongo_client = ""
     db = ""
-    column = ""
+    annotation_column = ""
+    annotation_detail_column = ""
     articles = []
-    
+    article_details={}
+    search_result_list=[]
```

Comment:
Doesn't this conflict with "self.search_result_list" and why does this exist, is it necessary, can we remove it?

Similar with all other variables defined under class but are not class variables. Such as "annotation_column", vs "self. annotation_column" and "annotation_detail_column" vs "self.annotation_detail_column" as well.